### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by ehoutland**
Adding test cases to deltaccd.

### CODE REVIEW

**Comment by JSchoenbachler** [comment]
Looking over this it all looks good! The only issues I have are minor ones, but here they are:

1. https://github.com/hugheylab/deltaccd/blob/653c97e4d52843d4c670bd80a76eddf4c82519c7/tests/testthat/test_non_exported.R#L85 This line is giving a warning because the function call `calcDeltaCCDSimple` in `expect_equal` returns a warning. Consider wrapping the function in `suppressWarnings` function call to prevent this from happening.
2.  I'm not particularly crazy about using the `{package}::{function}` in this. I would just use `library` to add the ggplot2 and vdiffr packages to the test runs, probably in each script since there are only 2 scripts and additional packages called.

Other than that, looks great!

**Review by JSchoenbachler — APPROVED** [review]
As stated in other comment, looks good, only suggesting 2 minor changes!

**Comment by jakejh** [comment]
I agree with Josh's suggestions, and I would go further to say that `suppressWarnings` should never be necessary in a test.

File: tests/testthat/test_non_exported.R [hunk 1]
```diff
@@ -112,6 +115,6 @@ test_that('plotHeatmapSimple', {
   cLims = calcColorLimits(cors$rho)
 
   p = plotHeatmapSimple(
-    ggplot2::ggplot(cors) + ggplot2::facet_wrap(ggplot2::vars(group)), cLims)
-  vdiffr::expect_doppelganger('basic heatmap', p)
+    ggplot(cors) + ggplot2::facet_wrap(vars(group)), cLims)
```
Comment (jakejh) [inline tests/testthat/test_non_exported.R]: Missed this `ggplot2::`

**Comment by jakejh** [comment]
I just noticed the NEWS.md file has a remnant of a merge conflict. Fix that and I think we're good to go.

**Pull request merged**